app/lib/encoding/slip32.ts

```diff
@@ -79,9 +88,9 @@ export const importKeyFromSlip32 = (slip32: string): {
   extendedKey: Key.ExtendedKey<PrivateKey.PrivateKey> | Key.ExtendedKey<PublicKey.PublicKey>
 } => {
   // Decode slip32
-  const {prefix, words}: { prefix: string, words: Buffer } = decode(slip32)
+  const {hrp, words}: { hrp: string, words: Buffer } = decode(slip32)
   // Check hrp
-  if ((prefix !== "xprv") && (prefix !== "xpub")) {
+  if ((hrp !== "xprv") && (hrp !== "xpub")) {
```

Comment:
Side comment (no changes required here): on the long term, we could define if we favor conjunction of negation `a !== b && a !== c` or negation of disjunction `!(a == b || a == c)` unsing [De Morgan's laws][demorgan] to improve readability of some logic assertions with tricky semantics.

In this case, I think it's just OK as is (we save one operator).

This is also equivalent to `!["xpriv", "xpub"].includes(hrp)`, which can be cool when you need to match against a lot of different invariants, but it has a serious performance price.

[demorgan]: https://en.wikipedia.org/wiki/De_Morgan%27s_laws